app/src/Homepage/Homepage.tsx

```diff
@@ -157,6 +157,52 @@ export const HomePage: React.FC = () => {
     );
 }
 
+export const ClassRating: React.FC<{category: string, rating: number, type: string}> = ({category, rating, type}) => {
+  const dynamicclassname = `rating-pair-rating ratingbox-${type}`;
```

Comment:
Can you make this camelCased? I think that's what the style guide tells us

Reply:
Sure. Actually just copied this from Thao's homepage code so I'll change her's too.